src/templates/CheckOut/checkout.scss

```diff
@@ -72,6 +72,10 @@ $mobile-layout-padding: ru(1);
 
   &-price {
     display: none;
+
+    small {
```

Comment:
should we just use `Text` component for this rather than the `small` html node? just seems like an anti-pattern to use an arbitrary percentage for font size rather than our brand sizes